README.adoc

```diff
@@ -198,17 +172,25 @@ include::{common-includes}/mvnbuild.adoc[]
 
 When the server is running, point your browser to the
 `http://localhost:9080/inventory/systems/localhost` URL.
-You receive a result in JSON format with the system properties of your local JVM. Next, point your
-browser to the `http://localhost:9080/system/properties` URL, which retrieves the information for a
-specific host. Notice that the results from the two URLs are identical.
+You receive the system properties of your local JVM from the `inventory` service. Next, point your
+browser to the the `system` service URL, which is `http://localhost:9080/system/properties`,
+to retrieve the system properties for the specific localhost.
+Notice that the results from the two URLs are identical because `inventory` service gets its results from
```

Comment:
I'd add "the" in front of `inventory` service.

Reply:
Should have the previous "the" here. Fixed! 👍